server/neptune/workflows/internal/deploy/workflow.go

```diff
@@ -88,8 +88,8 @@ func newRunner(ctx workflow.Context, request Request, tfWorkflow terraform.Workf
 	// can potentially change a root configuration
 	root := root.Root{
 		Name:      request.Root.Name,
-		Apply:     job.Job{Steps: convertToInternalSteps(request.Root.Apply.Steps)},
```

Comment:
nit: can we add new lines to make this more readable?

Reply:
sure can do that.